### PR TITLE
add geo3al data

### PR DESCRIPTION
**Pull request by CovMat**

### CODE REVIEW

File: source/dataset-CN/geo3al/geo3al.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+geo3al: 中国大陆及临区地质图数据
+================================
+
+:贡献者: 徐弥坚
+:原文链接: https://blog.xumijian.me/post/gmt-geo/
+
```
Comment (core-man) [inline source/dataset-CN/geo3al/geo3al.rst:6]:
```suggestion

----

```

**Comment by core-man** [comment]
@CovMat BTW, we now have only a simple contributing guide in https://docs.gmt-china.org/latest/contributing/, which will be further polished since we now have a few active contributors. But, please refer to https://seismo-learn.org/contributing/pull-request/ first about how to submit a friendly PR.

**Comment by CovMat** [comment]
> @CovMat Thanks for your hard work. We now have only a simple contributing guide in https://docs.gmt-china.org/latest/contributing/, which will be further polished since we now have a few active contributors. But, please refer to https://seismo-learn.org/contributing/pull-request/ first about how to submit a friendly PR. This will make PR-review friendly and your work could be accepted in a quick time.

这次应该都是按要求操作的

**Comment by core-man** [comment]
> 这次应该都是按要求操作的

Yes, but please refer to those guides so that we could work better as a team. . For example, capitalize the subject line as suggested in [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).

**Comment by CovMat** [comment]
> > 这次应该都是按要求操作的
> 
> Yes, but please refer to those guides so that we could work better as a team. . For example, capitalize the subject line as suggested in [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).

哦哦，了解

**Pull request closed without merging**